src/axom/slic/docs/sphinx/sections/getting_started.rst

```diff
@@ -0,0 +1,243 @@
+the desired :ref:`logMessageLevel` by calling ``slic::setLoggingMsgLevel()``.
+This provides another knob that the application can use to filter the type and
+level of messages to be captured.
+
+All log messages with the specified severity level or higher are captured.
+For example, the following code snippet sets the severity level to *debug*.
+
+.. literalinclude:: ../../../examples/basic/logging.cpp
+   :start-after: SPHINX_SLIC_SET_SEVERITY_BEGIN
+   :end-before: SPHINX_SLIC_SET_SEVERITY_END
+   :language: C++
+   :linenos:
+
+This indicates that all log messages that are *debug* or higher
+are captured otherwise, the messages are ignored. Since *debug* is the lowest
+severity level, all messages will be captured in this case.
+
+.. _slicExampleStep5:
+
+Step 5: Register a Log Stream
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Log messages can have one or more output destination. The output destination
+is specified by registering a corresponding :ref:`logStream` object to each
+:ref:`logMessageLevel`.
+
+The following code snippet uses the :ref:`GenericOutputStream` object,
+one of the :ref:`NativeLogStreams` provided by Slic, to specify ``std::cout``
+as the output destination for messages at each :ref:`logMessageLevel`.
+
+.. literalinclude:: ../../../examples/basic/logging.cpp
+   :start-after: SPHINX_SLIC_SET_STREAM_BEGIN
+   :end-before: SPHINX_SLIC_SET_STREAM_END
+   :language: C++
+   :linenos:
+
+.. note::
+
+   Instead of calling ``slic::addStreamToAllMsgLevels()`` an application
+   may use ``slic::addStreamToMsgLevel()`` that allows more fine grain
+   control of how to bind :ref:`logStream` objects to each
+   :ref:`logMessageLevel`. Consult the `Slic Doxygen API Documentation`_
+   for more information.
+
+The :ref:`GenericOutputStream`,  takes two arguments in its constructor:
+
+* A C++ ``std::ostream`` object that specifies the destination
+  of messages. Consequently, output of messages can be directed to the console,
+  by passing ``std::cout`` or ``std::cerr``, or to a file by passing a C++
+  ``std::ofstream`` object. In this case, ``std::cout`` is specified as the
+  output destination.
+
+* A string corresponding to the :ref:`logMessageFormat`, discussed in
+  :ref:`slicExampleStep3`.
+
+.. note::
+
+   Slic maintains ownership of all registered :ref:`logStream` instances and
+   will deallocate them when ``slic::finalize()`` is called.
+
+Step 5: Log Messages
+^^^^^^^^^^^^^^^^^^^^^
+
+Once the output destination of messages is specified, messages can be logged
+using the :ref:`sections/slic_macros`, as demonstrated in the code snippet
+below.
+
+.. literalinclude:: ../../../examples/basic/logging.cpp
+   :start-after: SPHINX_SLIC_LOG_MESSAGES_BEGIN
+   :end-before: SPHINX_SLIC_LOG_MESSAGES_END
+   :language: C++
+   :linenos:
+
+.. note::
+
+   By default, ``SLIC_ERROR()`` will print the specified message and a stacktrace
+   to the corresponding output destination and call ``abort()``, which aborts
```

Comment:
Does it always call abort()? I assume that for multiprocess MPI runs, it calls MPI_abort()?